File: Python/7py 3/to_jaden_case.py
"""
|--------------------------------------------------------------------------
| Jaden Casing Strings
|--------------------------------------------------------------------------
|
"""

def to_jaden_case(string):
   arr = string.split(" ")
   res = []
   for i in range(len(arr)):
      res.append((arr[i][0]).upper() + arr[i][1:])
   return ' '.join(res)


def toJadenCase(string):        
   return " ".join(w.capitalize() for w in string.split())


import string
def toJadenCase(NonJadenStrings):
   return string.capwords(NonJadenStrings)


quote = "How can mirrors be real if our eyes aren't real"
print(to_jaden_case(quote), "How Can Mirrors Be Real If Our Eyes Aren't Real")

File: PHP/7php 4/toJadenCase.php
<?php
/*
|--------------------------------------------------------------------------
| Jaden Casing Strings
|--------------------------------------------------------------------------
|
*/

function toJadenCase($string) 
{
   $arr = explode(' ', $string);
   $result = [];
   foreach ($arr as $i) {
      array_push($result, strtoupper($i[0]). substr($i, 1));
   }
   return implode(' ', $result);
}


function toJadenCase($string) 
{
   return ucwords($string);
}


function toJadenCase($str) 
{
   return join(' ', array_map('ucfirst', explode(' ', $str)));
}


$str = "How can mirrors be real if our eyes aren't real";
print_r(toJadenCase($str). " How Can Mirrors Be Real If Our Eyes Aren't Real");